<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item row="6" column="0">
         <widget class="QLabel" name="snrLabel">
          <property name="text">
           <string>Acceptable SNR (%)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>snrEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="nchansLabel">
          <property name="text">
           <string>Number of Subbands</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>nchansEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="widthLabel">
          <property name="text">
           <string>Pulse Width (ms)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>widthEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="tsampLabel">
          <property name="text">
           <string>Sampling Time (ms)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>tsampEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="bwLabel">
          <property name="text">
           <string>Channel width (MHz)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>bandwidthEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="highFreqLabel">
          <property name="text">
           <string>Tob Channel number</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>highFreqEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="highFreqEdit">
          <property name="text">
           <string>240</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="bandwidthEdit">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="tsampEdit">
          <property name="text">
           <string>0.005</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="widthEdit">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="nchansEdit">
          <property name="text">
           <string>31</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="snrEdit">
          <property name="text">
           <string>95</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="smearLabel">
          <property name="text">
           <string>Maximum Smearing (%)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
          <property name="buddy">
           <cstring>smearEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="smearEdit">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="scatteringLabel">
          <property name="text">
           <string>Acceptable Scattering (%)</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
          <property name="indent">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="scatteringEdit">
          <property name="text">
           <string>500</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="calcButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="saveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTabWidget" name="textTabWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>MDSM Parameters</string>
           </attribute>
           <widget class="QTableWidget" name="tableWidget">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>-10</x>
              <y>10</y>
              <width>381</width>
              <height>221</height>
             </rect>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <property name="rowCount">
             <number>7</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>175</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>27</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>21</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>27</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>21</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>175</number>
            </attribute>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <row>
             <property name="text">
              <string/>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Parameter</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Value</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string/>
             </property>
            </item>
           </widget>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Presto Parameters</string>
           </attribute>
           <widget class="QTableWidget" name="prestoTableWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>321</width>
              <height>231</height>
             </rect>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
